biscuit-auth/Cargo.toml

```diff
@@ -50,6 +50,10 @@ biscuit-parser = { version = "0.1.2", path = "../biscuit-parser" }
 biscuit-quote = { version = "0.2.2", optional = true, path = "../biscuit-quote" }
 chrono = { version = "0.4.26", optional = true, default-features = false, features = ["serde"] }
 serde_json = "1.0.117"
+ecdsa = { version = "0.16.9", features = ["signing", "verifying", "pem", "alloc", "pkcs8", "serde"] }
```

Comment:
should we feature-gate ecdsa support?

Reply:
we can, but what would it bring? Reducing the number of dependencies?